scripts/mesoscale/plots/exevs_mesoscale_atmos_sref_precip_spatial_plots.sh

```diff
@@ -69,16 +68,17 @@ python $USHevs/mesoscale/ush_sref_plot_precip_py/sref_atmos_plots.py
 
 cd $DATA/grid2grid_plots/plot_output/atmos.${VDATE}/precip/SL1L2_FBAR_24hrAccumMaps_CONUS_precip_spatial_map/images
 
-mask=`echo $vx_mask | tr '[A-Z]' '[a-z]'`
-for model in $model_list ; do
-  for fhr in 24 48 72 ; do
-    past=`$NDATE -$fhr ${VDATE}12`
-    INITDATE=${past:0:8}    
-    mv ${model}.v${VDATE}12.0${fhr}h.${mask}.png  evs.${model}.spatial_map.apcp_a24.init${INITDATE}.vlid${VDATE}12.f${fhr}.buk_${mask}.png
-  done
-done  
-
-tar -cvf evs.plots.sref.precip.spatial.map.v${VDATE}.tar *.png
+#mask=`echo $vx_mask | tr '[A-Z]' '[a-z]'`
+#for model in $model_list ; do
+#  for fhr in 24 48 72 ; do
+#    past=`$NDATE -$fhr ${VDATE}12`
+#    INITDATE=${past:0:8}    
+#    mv ${model}.v${VDATE}12.0${fhr}h.${mask}.png  evs.${model}.spatial_map.apcp_a24.vlid12z.f0${fhr}.${mask}.png
+#  done
+#done  
+
+
+tar -cvf evs.plots.sref.precip.spatial.map.v${VDATE}.tar *.gif
 
```

Comment:
@BinbinZhou-NOAA I just wanted to check on this file.
I wanted to make sure the lines 71-78 are meant to be commented out?
I wanted to make sure line 81 is meant to be changed from png to gif?

Reply:
Ah ok! Thanks @BinbinZhou-NOAA  for explaining the png to gif change in your previous comment!